cmd/create.go

```diff
@@ -71,37 +72,69 @@ Por padrão, o deployment embutido no binário é utilizado.`,
 				fmt.Println("\nE inicie o serviço:")
 				fmt.Println("   sudo systemctl enable docker")
 				fmt.Println("   sudo systemctl start docker")
+			}
+			if runtime.GOOS == "darwin" && containerEngine == "podman" {
+				// macOS podman
+				fmt.Println("\n📦 Para macOS, recomendamos Podman:")
+				fmt.Println("1. Instale o Homebrew caso não tenha:")
+				fmt.Println("   /bin/bash -c \"$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)\"")
+				fmt.Println("2. Instale o Podman")
+				fmt.Println("   brew install podman")
+				fmt.Println("3. Inicie o Podman:")
+				fmt.Println("   podman machine init")
+				fmt.Println("   podman machine start")
+			} else if runtime.GOOS == "linux" && containerEngine == "podman" {
+				// Linux podman
+				fmt.Println("\n📦 Para Linux, use o script de instalação oficial:")
+				fmt.Println("   curl -fsSL https://get.docker.com | bash")
+				fmt.Println("\nE inicie o serviço:")
+				fmt.Println("   sudo systemctl enable podman")
+				fmt.Println("   sudo systemctl start start")
```

Comment:
Acho que com a atualização do branch o typo retornou!

```suggestion
				fmt.Println("   sudo systemctl start podman")
```

Reply:
Ack